src/store/reducers/cohortBuilder.js

```diff
@@ -55,6 +73,8 @@ export default (state = initialState, action) => {
       return {
         ...state,
         ...action.payload,
+        dirty: false,
+        areSqonsEmpty: false,
```

Comment:
@jberube If we reset the virtual study... shouldn't the sqons be empty?

Reply:
Yeah, but that is success, not reset.